app/models/rook.rb

```diff
@@ -1,2 +1,6 @@
 class Rook < Piece
+  def piece_rules?(to_x:, to_y:)
+    return true if to_y == y_position || to_x == x_position
```

Comment:
Won't this just return true if the position we want to move it to (the target coords) are the same as the current position in either the x or y direction?

Maybe like 
```
return true if to_y > y_position || to_y < y_position && to_x == x_postion

```